emby/docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: emby_server_1
+      APP_PORT: 8096
```

Disable proxy auth, otherwise apps cannot connect.
```suggestion
      APP_PORT: 8096
      PROXY_AUTH_ADD: "false"
```